### PR TITLE
Chore/fullstack deploy

### PR DESCRIPTION
**Pull request by djavorek**
Required changes to be able to auto deploy main branch to AWS Lightsail

### CODE REVIEW

File: database/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,7 @@
+FROM mysql:8.0.29
+
+ENV MYSQL_DATABASE veryrezsi
+
+# All scripts in docker-entrypoint-initdb.d/ are automatically
+# executed during container startup
+COPY ./scripts/ /docker-entrypoint-initdb.d/
```
Comment (szattila98) [inline database/Dockerfile:7]: The law requiring a new line at the end of a file extends to both mortal and divine realms.

File: docker-compose.yml [hunk 2]
```diff
@@ -17,8 +17,8 @@ services:
       timeout: 20s
       retries: 10
 
-  mailhog:
-    container_name: mailhog
+  smtp:
+    container_name: veryrezsi_smtp
```
Comment (szattila98) [inline docker-compose.yml:21]: If the container name is no longer mailhog, then the docker config files should reflect that, as the DNS host resolve is based upon this.

File: server/core/src/email.rs [hunk 3]
```diff
@@ -59,7 +70,7 @@ where
     <T as AsyncTransport>::Error: std::fmt::Debug,
 {
     let to = to.parse().expect("to should be a valid email address");
-    let from = "Veryrezsi <noreply@veryrezsi.com>"
+    let from = "Veryrezsi <noreply@dev.veryrezsi.cloud>"
```
Comment (szattila98) [inline server/core/src/email.rs:73]: Should this address be configurable?

File: server/docker/config/app-config.toml [hunk 4]
```diff
@@ -6,5 +6,5 @@ log_level = "info"
 [mail_config]
 smtp_address = "mailhog"
```
Comment (szattila98) [inline server/docker/config/app-config.toml:7]: `smtp_address` should not be 'mailhog' if the container name changed. This won't work like this, I think.

**Review by szattila98 — CHANGES_REQUESTED** [review]

File: server/core/src/email.rs (hunk 3 above)
Comment (szattila98) [reply server/core/src/email.rs:73]: I will record it as an implementation detail!

Comment (djavorek) [reply server/core/src/email.rs:73]: Thanks

File: .github/workflows/database.yml [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+      - uses: actions/checkout@master
+      - uses: yokawasa/action-sqlcheck@v1.4.0
+        id: sqlcheck
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          directories: |
+            database/scripts
+      - name: Get output
+        run: echo "Issues found in previous step"
+        if: steps.sqlcheck.outputs.issue-found
```
Comment (szattila98) [inline .github/workflows/database.yml:27]: nice touch!

**Review by szattila98 — APPROVED** [review]
LGTM

**Pull request merged**